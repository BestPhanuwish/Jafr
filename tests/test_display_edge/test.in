a
0
7
666
6 6 6
dsn6bas
6