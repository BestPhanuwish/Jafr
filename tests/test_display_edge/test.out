Just a friendly reminder! You have these tasks to finish today.
- Due Due
- complete not complete complete

These tasks need to be finished in the next three days!
- indentation incident by 02/08/23

You have the following meetings today!
- should be today's meeting at 23:59

You have the following meetings scheduled over the next week!
- should be tomorrow's meeting on 02/08/23 at 00:01
- theoritically tomorrow's meeting on 02/08/23 at 00:00

What would you like to do?
1. Complete tasks
2. Add a new meeting.
3. Share a task.
4. Share a meeting.
5. Change Jafr's master directory.
6. Exit
Please pick number between 1-6
Please pick number between 1-6
Please pick number between 1-6
Please pick number between 1-6
Please pick number between 1-6
Please pick number between 1-6
